Azure-Samples/Cognitive-Services-Voice-Assistant
# Cofogg/fix above lock activation

## Purpose
Changes above lock from "close on lock" behavior to "create a new view above lock on above lock activation and close the views above and below lock upon dismissing the view above lock". Need to do additional work to sync the above and below lock views and maintain the below lock view on dismissing the above lock app.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
Launch below lock, lock the screen, then unlock. The app should still be running
Launch below lock, lock the screen, then activate above lock. The app should activate above lock.
After launching above lock, click the x button to dismiss the view. After unlocking, the below lock instance will be gone. This is expected.
